tests/e2e/testdata/download_releases.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+#if [ $# -ne 1 ]; then
+#  echo "Usage: ./download_releases.sh RELEASE_TAG"
+#  exit 1
+#fi
+echo "WIP: copy from local built instead of downloading"
```

Comment:
This works on my box. Happy to download from mesh-security repo soon

Reply:
You can!

https://github.com/osmosis-labs/mesh-security/releases/tag/v0.2.0-alpha.3

Just store the tag as a variable and swap out for v0.2.0 when we tag that